app/services/albums.js

```diff
@@ -0,0 +1,28 @@
+const request = require('request-promise');
+const errors = require('errors.js');
```

Comment:
Reference relative path of errors like `../errors`, avoid `.js` extension is not necessary